lib/views/custom_widgets/photo_collage.dart

```diff
@@ -138,75 +137,77 @@ class PhotoCollageState extends State<PhotoCollage> with UiLoggy {
 
   @override
   Widget build(BuildContext context) {
-    return Screenshot(
-      controller: screenshotController,
-      child: SizedBox(
-        height: 1000 + 2*widget.padding,
-        width: 1000*widget.aspectRatio + 2*widget.padding,
-        child: Observer(builder: (context) => _getLayout(AppLocalizations.of(context)!)),
+    Widget collageBox = FittedBox(
+      child: Screenshot(
+        controller: screenshotController,
+        child: SizedBox(
+          height: 1000 + 2 * widget.padding,
+          width: 1000 * widget.aspectRatio + 2 * widget.padding,
+          child: Observer(builder: (context) => _getLayout(AppLocalizations.of(context)!)),
+        ),
       ),
     );
+
+    if (widget.isVisible) return collageBox;
+
+    // We have already tested built in Flutter widgets like Visibility, Opacity, Offstage, but none of them work...
+    return Transform.translate(
+      offset: Offset(MediaQuery.sizeOf(context).width, 0),
+      child: collageBox,
+    );
```

Comment:
Still a ghetto way to achieve it of course, but elegant from the outside 😅

Reply:
We def should check whether Canvas suits our needs...